### PR TITLE
Update to 2.9.0

### PR DESCRIPTION
**Pull request by StarsDream00**
https://github.com/LiteLDev/LiteLoaderBDS/pull/967

### CODE REVIEW

**Comment by Futrime** [comment]
Does it pass compilation?

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
```
Comment (Futrime) [inline .github/FUNDING.yml:1]: Funding information should only be placed in the main repository.

**Comment by StarsDream00** [comment]
> Does it pass compilation?

It's from https://github.com/LiteLDev/LiteLoaderBDS/tree/develop/ScriptEngine

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,109 +1,77 @@
 cmake_minimum_required(VERSION 3.21)
 
-project(LLScriptEngine)
+if ("${LLSE_BACKEND}" STREQUAL "")
```
Comment (Futrime) [inline CMakeLists.txt:3]: Note that when splitting the script engine as a standalone module, this list file would be run independently. Therefore, these variables will not be accessed.

**Review by Futrime — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (StarsDream00) [reply CMakeLists.txt:3]: 这将会以子模块形式并入主仓库编译

**Comment by Futrime** [comment]
We hope that Script Engine, as a sub-module, can be compiled independently. This repository is created only for this purpose. Otherwise, it's no need to split Script Engine.

**Comment by StarsDream00** [comment]
> We hope that Script Engine, as a sub-module, can be compiled independently. This repository is created only for this purpose. Otherwise, it's no need to split Script Engine.

那PermissionAPI和ParticleAPI呢？

**Comment by Futrime** [comment]
@StarsDream00 那两个先不管，llse拆出来的目的只是做一个独立编译的实验

**Pull request closed without merging**